engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -49,36 +49,42 @@ struct avengers_shield_base_t : public paladin_spell_t
     }
     may_crit = true;
 
+
+    //Turn off chaining if focused enmity
     aoe = data().effectN( 1 ).chain_target();
     if ( p -> azerite.soaring_shield.enabled() )
     {
       aoe = as<int>( p -> azerite.soaring_shield.spell() -> effectN( 2 ).base_value() );
     }
 
-    // First Avenger hits +2 targets
-    aoe += as<int>( p -> talents.first_avenger -> effectN( 1 ).base_value() );
+    // Soaring Shield hits +2 targets
+    aoe += as<int>( p -> talents.soaring_shield -> effectN( 1 ).base_value() );
   }
 
   void impact( action_state_t* s ) override
   {
     paladin_spell_t::impact( s );
+    if ( p()->talents.tyrs_enforcer->ok() )
+    {
+      p()->trigger_tyrs_enforcer(s);
+    }
 
     if ( p() -> azerite.soaring_shield.enabled() )
     {
       p() -> buffs.soaring_shield -> trigger();
     }
 
-    // First Avenger absorb shield. Amount is additive per hit.
-    if ( p() -> talents.first_avenger -> ok() )
+    //Bulwark of Order absorb shield. Amount is additive per hit.
+    if ( p() -> talents.bulwark_of_order -> ok() )
     {
-      //First Avenger is capped at 30% of max hp. Hardcoded here.
+      //Bulwark of Order is capped at 30% of max hp. Hardcoded here.
```

Comment:
I can't see anything for this in spelldata, I assume testing in game synced up with 30% max?